CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # OMC3 Changelog
 
+#### 2024-11-21 - v0.20.3 - _jdilly_
+
+- Fixed:
+  - `analyse_dpp` issues in `beta_from_amplitude`, `chromatic_beating` and `dispersion`.
```

Comment:
Yeah, but.. basically that they were not taken into account, resulted in weird results or just straight out crashed under certain circumstances....

Reply:
I think it would be nice to write: fixed `analyse_dpp` parameter was not taken in consideration in ... for the future. It might not appear obvious to someone else from a git blame.